Sources/TuistLoader/Loaders/ManifestModelConverter.swift

```diff
@@ -98,12 +98,13 @@ public final class ManifestModelConverter: ManifestModelConverting {
             uniqueKeysWithValues: manifest.externalProjects
                 .map { project in
                     let projectPath = try AbsolutePath(validating: project.key.pathString)
+                    let isExternal = projectPath.components.contains(Constants.SwiftPackageManager.packageCheckoutDirectoryName)
```

Comment:
What do you think of encoding the value into the [projects](https://github.com/tuist/tuist/blob/main/Sources/TuistCore/Graph/DependenciesGraph.swift#L10) variable of the `DependenciesGraph` during its [initial load](https://github.com/tuist/tuist/blob/main/Sources/TuistLoader/Loaders/SwiftPackageManagerGraphLoader.swift#L152-L166)?

We should try to avoid these assumptions as much as possible.

Reply:
I've resolved it in a way that I think is semantically appropriate.